eventservice/events.yaml

```diff
@@ -25,7 +25,7 @@ paths:
         Auth: Requires a valid API Key or ID Token for access.
       security:
         - ApiKeyAuth: []
-        - UserAuth: []
+        - UserAndCoreTokenAuth: []
```

Comment:
The previous UserAuth verifies users' access to an event using their Rokwire group membership. Users can view only public events and private group events that they have access to due to Rokwire Groups BB group memberships. Will that be affected by this new change? Same question for all other Events BB endpoints that replaces UserAuth with UserAndCoreTokenAuth.

Reply:
Currently, the groups BB only supports those checks for shibboleth tokens. We've added some changes [here](https://github.com/rokwire/rokwire-building-blocks-api/blob/ac52faff857dc3d52d2ec4e45e3bef4d62f2c1a1/eventservice/api/utils/group_auth.py#L40-L44) to extend those checks to the core tokens as well. But additional changes need to be made on the groups BB as well.